addons/io_hubs_addon/nodes/lightmap.py

```diff
@@ -18,6 +18,20 @@ def poll(cls, context):
 ]
 
 
+def add_node_menu_blender4(self, context):
+    self.layout.menu("NODE_MT_mozilla_hubs_nodes")
+
+
+class NODE_MT_mozilla_hubs_nodes(bpy.types.Menu):
+    """Add node menu for Blender 4.x"""
+    bl_label = "Hubs"
+    bl_idname = "NODE_MT_mozilla_hubs_nodes"
+
+    def draw(self, context):
+        layout = self.layout
+        layout.operator("node.add_node", text="MOZ_lightmap settings").type = "moz_lightmap.node"
```

Comment:
I think the `node.add_node` operator should use the `use_transform` option set to True so that it maintains the same behavior as in Blender 3.6